kiro-cli/README.md

```diff
@@ -0,0 +1,67 @@
+# Kiro AI Assistant for OpenSearch Migrations
+
+Configuration files for [Kiro](https://kiro.dev), an AI-powered development assistant for OpenSearch Migrations.
+
+## Quick Start (Gradle)
+
+```bash
+# Run kiro chat with the opensearch-migration agent
+./gradlew :kiro-cli:kiroAgent
+
+# Run kiro chat without a specific agent
+./gradlew :kiro-cli:kiroChat
+
+# Create kiro-assistant.tar.gz for distribution
+./gradlew :kiro-cli:packageKiro
+```
+
+### With Audit Logging
+
+```bash
+script -q ~/kiro-audit/$(date +%Y%m%d-%H%M%S).log ./gradlew :kiro-cli:kiroAgent
+```
+
+## Download from Release
+
+Download `kiro-assistant.tar.gz` from the [latest release](https://github.com/opensearch-project/opensearch-migrations/releases) and extract:
```

Comment:
Why do you have directions on downloading the artifact within the README when a user can rebuild directly?